api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java

```diff
@@ -61,6 +61,27 @@
  *   <li>to cancel the LRA context when the method returns particular HTTP
  *       status codes</li>
  * </ul>
+ * 
+ * Possible return HTTP status codes are:
```

Comment:
Isn't this a business method. How can we stipulate what responses a business method should return?

Reply:
Through filters? This how we've defined everything so far. See for example https://github.com/eclipse/microprofile-lra/blob/master/api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java#L206.

Reply:
If we indeed do need to add anything extra then something like the following would be less confusing:
 
> When a method is about to execute in the context of an LRA but the LRA has already ended (and we add the appropriate caveat for nested LRAs) then the method is not executed and a 410 response code is returned.

And if we are not already covered for the 412 case then we should fix the relevant part of the javadoc with something like:

> When an method is about to execute in the context of an LRA and XXX/whatever is in an invalid state then the method is not executed and a 412 response code is returned.

But that is confusing too so we should we would need to be explicit.

Reply:
fair enough. I will add something similar to this PR. But I won't change the already specified 412 elsewhere in the text. I will do so when I will be doing the final sweep of the specification.